cmd/server/main.go

```diff
@@ -164,6 +164,29 @@ func cacheConfigBuilder() *authorizer.SystemCache {
 	return authorizer.NewSystemCache(config, make(chan struct{}))
 }
 
+type logger struct {
+	printFn func(template string, args ...interface{})
+}
+
+func (l logger) Printf(template string, args ...interface{}) {
+	l.printFn(template, args)
+}
+
+func createBackendConfig() (authorizer.BackendConfig, chan struct{}) {
```

Comment:
Is the channel returned in the 2nd parameter used anywhere?

Reply:
not yet 👻